packages/workbox-background-sync/test/browser/request-queue.js

```diff
@@ -30,6 +30,14 @@ describe('request-queue tests', () => {
     callbacks,
   });
 
+  const getDelayedPromise = () => {
+    return new Promise((resolve)=>{
+      setTimeout(()=>{
```

Comment:
Yes it does. We had the prettier discussion, but no one drove it through to be useful in our repo with our eslint config.

If you want to use that and can get it working, then feel free to add it. Until then, it's manual.